### PR TITLE
feat: add support for window versioning

### PR DESCRIPTION
**Pull request by irainia**
This is impacted by [this](https://github.com/odpf/optimus/issues/453) issue, as there is a breaking changes.

### CODE REVIEW

File: task/bq2bq/go.mod [hunk 1]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/googleapis/google-cloud-go-testing v0.0.0-20210719221736-1c9a4c676720
 	github.com/hashicorp/go-hclog v1.2.0
 	github.com/mitchellh/hashstructure/v2 v2.0.2
-	github.com/odpf/optimus v0.2.2-0.20220606050552-243f0c02d055
+	github.com/odpf/optimus v0.2.6-rc.1.0.20220726091918-0b5282570d77
```
Comment (sravankorumilli) [inline task/bq2bq/go.mod:11]: same here

File: .goreleaser.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 env:
-  - OPTIMUS_RELEASE_URL=https://github.com/odpf/optimus/releases/download/v0.2.0-alpha/optimus_0.2.0-alpha_linux_x86_64.tar.gz
+  - OPTIMUS_RELEASE_URL=https://github.com/odpf/optimus/releases/download/v0.2.6-rc.1/optimus_0.2.6-rc.1_linux_x86_64.tar.gz
```
Comment (sravankorumilli) [inline .goreleaser.yml:2]: lets update with the latest release before merging

**Review by sravankorumilli — CHANGES_REQUESTED** [review]

File: task/bq2bq/go.mod (hunk 1 above)
Comment (irainia) [reply task/bq2bq/go.mod]: updated. but, note that we are using the latest (at the time of writing) commit of the [window branch](https://github.com/odpf/optimus/tree/window-versioning).

File: .goreleaser.yml (hunk 2 above)
Comment (irainia) [reply .goreleaser.yml]: updated. however, we are just using the one in the main branch (of this repository)

**Pull request merged**